python/prophet/diagnostics.py

```diff
@@ -109,6 +107,10 @@ def map(self, func, *iterables):
                         for args in zip(*iterables)
                      ]
                      return results
+                     
+    disable_tqdm: if True it disables the progress bar that would otherwise show up when parallel=None
+    extra_output_columns: A String or List of Strings e.g. 'trend' or ['trend'].
```

Comment:
Hmm I don't think you've handled the String case. If the input is `trend` I think we'd be adding ['t', 'r', 'e', 'n', 'd'] to the output columns